zerver/openapi/zulip.yaml

```diff
@@ -11518,6 +11518,24 @@ paths:
                               type: string
                               description: |
                                 The name of the bot.
+                            display_name:
+                              type: string
+                              description: |
+                                The name of the bot that is displayed to the users.
```

Comment:
This documentation is super confusing to someone who doesn't know about this system. This line should clearly explain how this is different from `name`. E.g.  (please check this is correct, I'm going off memory):

> A machine-readable unique name uniquely identifying an official Zulip integration, all-lower-case without spaces.

and 

> A human-readable display name identifying an official Zulip integration that this bot implements.